internal/server/metrics.go

```diff
@@ -9,6 +9,13 @@ import (
 	"github.com/infrahq/infra/metrics"
 )
 
+var outboundRequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	Namespace: "http",
+	Name:      "outbound_request_duration_seconds",
+	Help:      "A histogram of outbound call durations made from the server to an external source, in seconds.",
+	Buckets:   prometheus.ExponentialBuckets(0.001, 2, 15),
```

Comment:
It's probably not necessary to start histogram buckets at 0.001s since outgoing requests will never take that little time. This also only produces 15 buckets so the largest bucket is only ~16s. If a request takes longer, it'll get put in to the `+Inf` bucket which doesn't tell us much.

Maybe start it at 0.1s which will also gives a max bucket size of ~1638s while still providing small enough buckets on the low end.